test/template-test.js

```diff
@@ -0,0 +1,37 @@
+import template from '../src/template'
+
+
+describe('template', () => {
+  it('execute', (done) => {
```

Comment:
`done` is only necessary if the test is asynchronous. You can remove it here and in the tests below.

Reply:
nit: In order to make the output readable, use:

 `it('should execute',` that way you can read it as `template should execute` (where the `it` is replaced by the previous `describe` and the rest can be read.